### PR TITLE
Add UnUniFi

### PR DESCRIPTION
**Pull request by Senna46**
Create a new entry for UnUniFi.
Website: https://ununifi.io/

Please review to merge it.
Our cosmos chain-registry is here https://github.com/cosmos/chain-registry/tree/master/ununifi

### CODE REVIEW

**Comment by Senna46** [comment]
@HeesungB The chainID is `ununifi-beta-1`, but it is the mainnet. It will be changed to `ununifi-1` in a hard fork soon. (Because we can't change it with on-chain upgrades)

**Comment by HeesungB** [comment]
@Senna46 Ok. Could you mention after change?

**Comment by Senna46** [comment]
@HeesungB From what I have confirmed, the Chain ID change will be a while in the future.
We would like to merge with this content first. Will change Chain ID later.

**Comment by HeesungB** [comment]
@Senna46
Your chainIdentifier is `ununifi-beta`.
So Could you change name `ununifi`to `ununifi-beta` and image directory name?

**Comment by Senna46** [comment]
Changed directory name
Is this OK?

**Comment by HeesungB** [comment]
@Senna46 Yes it is ok.
But, Could you rename `cosmos/ununifi.json` to `cosmos/ununifi-beta.json`?
and Could you remove `coinGeckoId`field about all currencies?
because `GUU` is not registered in coinGecko site.

**Comment by Senna46** [comment]
@HeesungB Changed. Now once we want to merge. 
I will create a Pull Request again when the beta is removed.

File: cosmos/ununifi-beta.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "rpc": "http://a.lcd.ununifi.cauchye.net:26657",
+  "rest": "https://a.lcd.ununifi.cauchye.net:1318",
+  "chainId": "ununifi-beta-v1",
+  "chainName": "UnUniFi",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "GUU",
+    "coinMinimalDenom": "uguu",
+    "coinDecimals": 6,
+    "coinGeckoId": "ununifi",
```
Comment (HeesungB) [inline cosmos/ununifi-beta.json:11]: @Senna46  It should be removed

Comment (HeesungB) [reply cosmos/ununifi-beta.json:11]: `coinGeckoId` field.

**Comment by HeesungB** [comment]
@Senna46 Sorry. My bad.
Your chainIdentfier is `ununifi-beta-v1` not `ununifi-beta`.
So Could you change file names?

**Comment by Senna46** [comment]
I've made the change, is that correct? @HeesungB

**Comment by HeesungB** [comment]
@Senna46 Yes It is correct.
Could you add 'node provider' information?

**Comment by Senna46** [comment]
@HeesungB I added nodePrivider field

File: cosmos/ununifi-beta-v1.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  "rpc": "http://a.lcd.ununifi.cauchye.net:26657",
+  "rest": "https://a.lcd.ununifi.cauchye.net:1318",
+  "nodeProvider": {
+    "name": "UnUniFi",
+    "email": "info@ununifi.io",
+    "website":"https://ununifi.io/"
+  },
+  "chainId": "ununifi-beta-v1",
+  "chainName": "UnUniFi",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi/chain.png",
```
Comment (HeesungB) [inline cosmos/ununifi-beta-v1.json:11]: @Senna46 
Link will be changed from `ununifi/chain.png` to `ununifi-beta-v1/chain.png`

File: cosmos/ununifi-beta-v1.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  "stakeCurrency": {
+    "coinDenom": "GUU",
+    "coinMinimalDenom": "uguu",
+    "coinDecimals": 6,
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi/uguu.png"
```
Comment (HeesungB) [inline cosmos/ununifi-beta-v1.json:16]: @Senna46

File: cosmos/ununifi-beta-v1.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "bech32PrefixValPub": "ununifivaloperpub",
+    "bech32PrefixConsAddr": "ununifivalcons",
+    "bech32PrefixConsPub": "ununifivalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "GUU",
+      "coinMinimalDenom": "uguu",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi-beta/uguu.png"
```
Comment (HeesungB) [inline cosmos/ununifi-beta-v1.json:36]: @Senna46

File: cosmos/ununifi-beta-v1.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "GUU",
+      "coinMinimalDenom": "uguu",
+      "coinDecimals": 6,
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi-beta/uguu.png",
```
Comment (HeesungB) [inline cosmos/ununifi-beta-v1.json:44]: @Senna46

**Comment by Senna46** [comment]
I fixed image link. @HeesungB

File: cosmos/ununifi-beta-v1.json [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    "website":"https://ununifi.io/"
+  },
+  "chainId": "ununifi-beta-v1",
+  "chainName": "UnUniFi",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/ununifi-beta-v1/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "GUU",
+    "coinMinimalDenom": "uguu",
+    "coinDecimals": 6,
+    "coinGeckoId": "ununifi",
```
Comment (HeesungB) [inline cosmos/ununifi-beta-v1.json:16]: @Senna46 Could you remove `coinGeckoId` field?

Comment (Senna46) [reply cosmos/ununifi-beta-v1.json]: Removed. I forgot.

**Comment by HeesungB** [comment]
@Senna46 Thank you for contribution!

**Pull request merged**